Observable/Classes/Observable+Binding.swift

```diff
@@ -0,0 +1,120 @@
+//
+// Observable+Binding.swift
+// Observable
+//
+// Created by bfrolicher on 20/07/2020.
+//
+import Foundation
+
+public extension Observable {
+    
+    /// Bind wrappedValue to `objectKeyPath` property of type `T` in object `O`
+    ///
+    /// - Parameter object: Owner of the property
+    /// - Parameter objectKeyPath: Keypath property
+    /// - Parameter disposal: Observable container
+    ///
+    /// Sample :
+    ///
+    /// ```
+    /// class SomeViewModel {
+    ///     var username =  Observable("")
+    /// }
+    ///
+    /// class ViewController {
+    ///     self.viewModel.username.bind(to: self.usernameLabel, \.text, disposal: &self.disposal)
+    /// }
+    ///
+    /// ```
+    ///
+    func bind<O>(to object: O, _ objectKeyPath: ReferenceWritableKeyPath<O, T>, disposal: inout Disposal) {
+        self.observe { [weak self] (new, old) in
+            guard let strongSelf = self else { return }
```

Comment:
What if we use `guard let self = self else { return }` here?

Reply:
It should work. 
A habit ^^

Reply:
I changed it to `self = self ` ✌️